### ISSUE TITLE
WalletConnect: personal_sign breaks on message with line break.

### ISSUE THREAD

**Issue by Benjythebee** [trustwallet/wallet-core#2148]
**Describe the bug**
Trying to implement login support of TrustWallet app via WalletConnect.
Our `personal_sign` method has a message similar to 
```
# Terms of Services

By signing, you agree to our terms and conditions ...
```

and it works fine on Walletconnect x Metamask and  Walleconnect x Gnosis Safe.

However, I noted it wasn't working for TrustWallet. 
By "wasn't working for TrustWallet" I mean that the transaction signature request never shows up on my phone while it does on Metamask and other.

I did some debugging and it seems that TrustWallet doesn't like a `personal_sign` with a message that has line-breaks (or multiple lines?).

**To Reproduce**

The easiest way to reproduce this is to **expose your provider to window** and interact with the provider in your console:

1. Call `await window.provider.enable()` to show the QRcode in Walletconnect.
2. Scan QrCode with TrustWallet to connect.
3. Now that you have the accounts, call this in your dev console:
```js
await provider.request({ method: 'personal_sign', params: [`# Terms Of Services
test`, provider.accounts[0]], jsonrpc: '2.0' })
```
Note there is a line break between the word `Services` and `test`. If you want to type your own message to test this, use SHIFT+Enter to create a line break.

4. See no Transaction signature request on your phone Trust wallet app.
5. Now try this:
```js
await provider.request({ method: 'personal_sign', params: [`# Terms Of Services`, provider.accounts[0]], jsonrpc: '2.0' })
```
6. A Transaction signature request will pop up in your phone and the request will eventually resolve.

**Expected behavior**

We expect `personal_sign` to behave consistently across providers. This means the method should work even with line breaks, just like on metamask.

The screenshot below shows the signing TX popping up on Metamask Mobile via WalletConnect. This screenshot is to prove that line breaks are supported on popular providers.
<img src="https://user-images.githubusercontent.com/38708022/163742748-43632867-7708-47fa-85db-70903705a534.png" width='200px'></img>

**Screenshots**

The screenshot below represents step 6 followed by step 3.
The first call has no Line break and returns the signature;
The second call has a line break and does nothing.
![image](https://user-images.githubusercontent.com/38708022/163741032-9f99b34b-d38e-47e3-a344-89b41298eecd.png)

**Additional context**
using:
    "@walletconnect/ethereum-provider": "^1.7.7"
and
    Trust Wallet Mobile - Android

**Comment by vcoolish** [comment]
Thank you for report! This issue will be fixed in next 5.33 version. This one is caused by google navigation component bug.
https://issuetracker.google.com/issues/217815060?hl=pt-br

**Comment by hewigovens** [comment]
Please update to latest version (5.34) on Google Play, should be fixed
